packages/ods/src/components/tooltip/src/index.html

```diff
@@ -97,6 +97,17 @@
 <!--      </p>-->
 <!--    </ods-tooltip>-->
 <!--  </ods-modal>-->
+  <p>Accessibility</p>
+  <ods-icon
+    aria-labelledby="tooltip-example-a11y-content"
+    id="tooltip-example-a11y-trigger"
+    name="circle-question" >
```

Comment:
Add `tabindex="0"` so that user can focus through keyboard navigation on the tooltip trigger